### ISSUE TITLE
ddd

### ISSUE THREAD

**Issue by Handumdum** [Handumdum/handumdum.github.com#1]
<!DOCTYPE html>
<html lang="ko">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
    <title>부트스트랩 101 템플릿</title>

    
    <link href="css/bootstrap.min.css" rel="stylesheet">

    
    
    
  </head>
  <body>
    <h1>Hello, world!</h1>
<nav class="navbar navbar-default">   <div class="container-fluid">          <div class="navbar-header">       <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">         <span class="sr-only">Toggle navigation</span>         <span class="icon-bar"></span>         <span class="icon-bar"></span>         <span class="icon-bar"></span>       </button>       <a class="navbar-brand" href="#">Brand</a>     </div>           <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">       <ul class="nav navbar-nav">         <li class="active"><a href="#">Link <span class="sr-only">(current)</span></a></li>         <li><a href="#">Link</a></li>         <li class="dropdown">           <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Dropdown <span class="caret"></span></a>           <ul class="dropdown-menu" role="menu">             <li><a href="#">Action</a></li>             <li><a href="#">Another action</a></li>             <li><a href="#">Something else here</a></li>             <li class="divider"></li>             <li><a href="#">Separated link</a></li>             <li class="divider"></li>             <li><a href="#">One more separated link</a></li>           </ul>         </li>       </ul>       <form class="navbar-form navbar-left" role="search">         <div class="form-group">           <input type="text" class="form-control" placeholder="Search">         </div>         <button type="submit" class="btn btn-default">Submit</button>       </form>       <ul class="nav navbar-nav navbar-right">         <li><a href="#">Link</a></li>         <li class="dropdown">           <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Dropdown <span class="caret"></span></a>           <ul class="dropdown-menu" role="menu">             <li><a href="#">Action</a></li>             <li><a href="#">Another action</a></li>             <li><a href="#">Something else here</a></li>             <li class="divider"></li>             <li><a href="#">Separated link</a></li>           </ul>         </li>       </ul>     </div>   </div> </nav>
<h1 style="text-align: center; font-style: Courier New;"> 기계공학부 20162725 한재원 </h1>
<p style="text-align: center; font-style: Courier New;"> <strong> 멋쟁이 사자처럼 </strong> 5기 이수 중 </p>
<hr></hr>
<center><img src="https://secure.static.tumblr.com/3cc524417222dfb4fa0026d531a8c71b/dw3na6s/IMgnshxp6/tumblr_static_tumblr_static_bklurccr580so84wc0wc4gkws_640.jpg"></center>
<p> </p>
<p> </p>
<center>
<p><span class="glyphicon glyphicon-search" aria-hidden="true"></span></p>
<p style="font-family: verdana;">about me</p>
<p> 중앙대학교 기계공학부 16학번 </p>
<p> 경기도 거주, 21살 </p>
<p> <em>멋쟁이 사자처럼</em> 5기 이수 중 </p>
<p>*</p>
</center>
<p></p>
<p></p>
<p></p>
<center><a class="btn btn-default" href="#" role="button">고체역학 및연습</a>
<button class="btn btn-default" type="submit">열역학 및 연습</button>
<button class="btn btn-default" type="submit">동역학 및 연습</button>
<button class="btn btn-default" type="submit">컴퓨터 응용</button>
<button class="btn btn-default" type="submit">통계학</button>
</center>
<p></p>
<p></p>
<p></p>
<p></p>
<center><p>   <button type="button" class="btn btn-primary btn-lg"> if you want to know about me more, </button>
<button type="button" class="btn btn-default btn-lg"> Send me an E-mail! </button> </p> </center>
<p></p>
<p></p>
<center>
<form class="form-inline">   
<div class="form-group">     
<label for="exampleInputName2">Name</label>     
<input type="text" class="form-control" id="exampleInputName2" placeholder="Jane Doe">   
</div>  
<div class="form-group">    
<label for="exampleInputEmail2">Email</label>  
<input type="email" class="form-control" id="exampleInputEmail2" placeholder="jane.doe@example.com"> 
</div>  
<button type="submit" class="btn btn-default">Send invitation</button>
</form>
</center>
<div style="width:50px; height:50px; background-color:white"></div>
    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
    
    <script src="js/bootstrap.min.js"></script>
  </body>
</html>
